README.md

```diff
@@ -13,97 +13,44 @@ SPDX-License-Identifier: Apache-2.0
     <a href="https://pkg.go.dev/mod/github.com/shipwright-io/build"> <img src="https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white"></a>
 </p>
 
-# Shipwright - a framework for building container images on Kubernetes
+# Shipwright
 
-Shipwright is an extensible framework for building container images on Kubernetes. With Shipwright,
-developers can define and reuse build strategies that build container images for their CI/CD
-pipelines. Any tool that builds images within a container can be supported, such
-as [Kaniko](https://github.com/GoogleContainerTools/kaniko),
-[Cloud Native Buildpacks](https://buildpacks.io/), and [Buildah](https://buildah.io/).
+Shipwright is an extensible framework for building container images on Kubernetes.
 
-## Dependencies
+## Why?
 
-| Dependency                                | Supported versions                     |
-| ----------------------------------------- | -------------------------------------- |
-| [Kubernetes](https://kubernetes.io/)      | v1.15.\*, v1.16.\*, v1.17.\*, v1.18.\* |
-| [Tekton](https://cloud.google.com/tekton) | v0.20.1                                |
+With Shipwright developers get a simplified approach for building container images, by defining a minimal YAML that does not require
+any previous knowledge of containers or containers tooling. All you need is your source code in git and access to a container registry.
 
-## Build Strategies
-
-The following [Build Strategies](docs/buildstrategies.md) are installed by default:
-
-* [Source-to-Image](docs/buildstrategies.md#source-to-image)
-* [Buildpacks-v3](docs/buildstrategies.md#buildpacks-v3)
-* [Buildah](docs/buildstrategies.md#buildah)
-* [Kaniko](docs/buildstrategies.md#kaniko)
-* [ko](docs/buildstrategies.md#ko)
-
-Users have the option to define their own `BuildStrategy` or `ClusterBuildStrategy` resources and make them available for consumption via the `Build` resource.
-
-## Custom Resources
-
-Shipwright defines four CRDs:
-
-* The `BuildStrategy` CRD and the `ClusterBuildStrategy` CRD is used to register a strategy.
-* The `Build` CRD is used to define a build configuration.
-* The `BuildRun` CRD is used to start the actually image build using a registered strategy.
+Shipwright can support any tool that can build container images, such as [Kaniko](https://github.com/GoogleContainerTools/kaniko),
```

Comment:
`Shipwright can support any tool that can build container images` -> `Shipwright supports several verified tools that can build container images` ? Or remove one `can` from the sentence